src/plugins/dashboard/server/saved_objects/dashboard_migrations.ts

```diff
@@ -237,12 +240,15 @@ export const createDashboardSavedObjectTypeMigrations = (
     '7.3.0': flow(migrations730),
     '7.9.3': flow(migrateMatchAllQuery),
     '7.11.0': flow(createExtractPanelReferencesMigration(deps)),
-    ...Object.fromEntries(embeddableMigrations),
+
+    ...embeddableMigrations,
 
     /**
      * Any dashboard saved object migrations that come after this point will have to be wary of
      * potentially overwriting embeddable migrations. An example of how to mitigate this follows:
      */
-    // '7.x': flow(yourNewMigrationFunction, embeddableMigrations['7.x'])
+    // '7.x': flow(yourNewMigrationFunction, embeddableMigrations['7.x'] ?? identity),
```

Comment:
@ThomThomson I've updated that code a little bit to make it working. Please have a look

Reply:
Yes if we do run into that situation, we won't be totally sure that the embeddable migration function will be defined for that version. Good thinking!